packages/client/src/Views/GameSetupNew.jsx

```diff
@@ -55,6 +56,11 @@ const AdminGameSetup = () => {
     channels
   } = data
   const tabs = Object.keys(data)
+
+  // filter out all legacy channels
+  const allChannels = channels.channels
+  channels.channels = allChannels.filter(channel => !channel.format || !LEGACY_CHANNEL.includes(channel.format))
```

Comment:
I'm going to remove the above block @lebaphi  - the correct fix is to remove (delete) channels in legacy formats. I'll (try) to modify the UI to show a "legacy channel" warning snippet when one is encountered. This will lead me/us to fix the data in the wargame definition.